phpBB/phpbb/db/migration/data/v320/remove_profilefield_aol.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\db\migration\data\v320;
+
+class remove_profilefield_aol extends \phpbb\db\migration\migration
+{
+	static public function depends_on()
+	{
+		return array(
+			'\phpbb\db\migration\data\v32x\v322',
+		);
+	}
+
+	public function update_schema()
+	{
+		return array(
+			'drop_columns'	=> array(
+				$this->table_prefix . 'profile_fields_data'			=> array(
+					'pf_phpbb_aol',
+				),
+			),
+		);
+	}
+
+	public function revert_schema()
+	{
+		return array(
+			'add_columns'	=> array(
+				$this->table_prefix . 'profile_fields_data'			=> array(
+					'pf_phpbb_aol'		=> array('VCHAR', ''),
+				),
+			),
+		);
+	}
+
+	public function update_data()
+	{
+		return array(
+			array('custom', array(array($this, 'delete_custom_profile_field_data'))),
+		);
+	}
+
+	public function revert_data()
+	{
+		return array(
+			array('custom', array(array($this, 'create_custom_field'))),
+		);
+	}
+
+	public function delete_custom_profile_field_data()
+	{
+        $sql = 'SELECT field_id
```

Comment:
Please don't use spaces to indent lines but rather spaces (see https://area51.phpbb.com/docs/32x/coding-guidelines.html#editorsettings). That is also one of the reasons why your build is failing on travis:
https://travis-ci.org/phpbb/phpbb/jobs/362237461#L630

Reply:
Okay yeah i though i setted up my IDE correctly, my bad :( . And about the commit message im not sure why my line endings are normal vim line endings, will try to see what is going on there maybe a HEX will clarify lol.